docs/src/pages/components/data-grid/filtering/filtering.md

```diff
@@ -230,9 +230,26 @@ const ratingColumnType: GridColTypeDef = {
 
 You can customize the rendering of the filter panel as shown in [the component section](/components/data-grid/components/#overriding-components) of the documentation.
 
+### Customize the filter panel content
+
+The customization of the filter panel content, can be performed by passing props to the he default `<GridFilterPanel/>` component.
+These props allows you to override
+
+- The available `linkOperators`
+- The order of the column selector
+- The style of the controller for each input
```

Comment:
So the way to modify input components would be the following

```js
<DataGrid 
	componentsProps={{
		filterFormDeleteIcon: { sx: /* ... */ },
		filterFormLinkOperatorInput: { sx: /* ... */, disabled: true, hiddenLabel: true },
		filterFormOperatorInput: { sx: /* ... */ },
		filterFormColumnInput: { sx: /* ... */ },
		filterFormValueInput: { sx: /* ... */ },
	}}
/>
```

If  yes, should we keep the className?

Reply:
I was expecting that we would support the following:

```tsx
<DataGrid 
  componentsProps={{
    filterPanel: {
      FilterFormProps: {
        deleteIconProps: { sx: /* ... */ },
        linkOperatorInputProps: { sx: /* ... */, disabled: true, hiddenLabel: true },
        operatorInputProps: { sx: /* ... */ },
        columnInputProps: { sx: /* ... */ },
        valueInputProps: { sx: /* ... */ },
      }
    }
  }}
/>
```

> If yes, should we keep the className?

I didn't understand. Which className?

Reply:
I'm talking about the CSS classes I introduced in this PR. The ones allowing to customize the filter panel as follow:

```js
<DataGrid
	...
	componentsProps={{
		filterPanel: {
			sx: {
				"& .MuiDataGrid-filterForm": { ... },
				"& .MuiDataGrid-closeIconController": { ... },
				...
			}
		}
	}}
/>
```

It is an alternative way to customize the filter panel so I can :
- revert the modifications, and we keep only one way to customize the filter panel inputs
- I rename classes such that they are similar to `FilterFormProps` keys. For example `.MuiDataGrid-closeIconController` would become `.MuiDataGrid-deleteIcon`.

Reply:
I would keep the class names, but scoping them to avoid too generic names, e.g. `.MuiDataGrid-deleteIcon`. We might have another `.MuiDataGrid-deleteIcon` in the future. As suggestion, all inner components of the virtualization have the `virtualScroller` prefix in the class name: https://github.com/mui-org/material-ui-x/blob/24cf3fbbe7907e8264f6e2fde73a82919a5c5dd0/packages/grid/_modules_/grid/components/virtualization/GridVirtualScrollerRenderZone.tsx#L15